deployment/CP01CNSP_Deploy.csv

```diff
@@ -1,10 +1,37 @@
+AT27,,AT27,,CP01CNSP-SP001-07-SPKIRJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-SPKIRJ-00265,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-08-CTDPFJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-CTDPFJ-00347,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-09-FLORTJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-FLORTJ-01205,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+AT27,,AT27,,CP01CNSP-SP001-10-PARADJ000,3,1,2015-05-13T12:55:00,2015-05-13T13:29:00,CGMCP-01CNSP-00003,CGVEH-CSPPCP-00005,CGINS-PARADJ-00437,40.139125,-70.779998,,101,133,Malfunction; recovered for corrective action
+SI20150722,,MGL1512,,CP01CNSP-SP001-00-SPPENG000,4,1,2015-07-23T17:20:00,2015-08-29T11:30:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGCON-CSPPCP-00005,40.13408,-70.77013,,101,134,Profiling body detached and on surface; recovered by SOOP
+SI20150722,,MGL1512,,CP01CNSP-SP001-02-OPTAAJ000,4,1,2015-07-23T17:20:00,2015-08-29T11:30:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-OPTAAJ-00181,40.13408,-70.77013,,101,134,Profiling body detached and on surface; recovered by SOOP
+SI20150722,,MGL1512,,CP01CNSP-SP001-03-NUTNRJ000,4,1,2015-07-23T17:20:00,2015-08-29T11:30:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-NUTNRJ-00428,40.13408,-70.77013,,101,134,Profiling body detached and on surface; recovered by SOOP
+SI20150722,,MGL1512,,CP01CNSP-SP001-05-VELPTJ000,4,1,2015-07-23T17:20:00,2015-08-29T11:30:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-VELPTJ-11915,40.13408,-70.77013,,101,134,Profiling body detached and on surface; recovered by SOOP
+SI20150722,,MGL1512,,CP01CNSP-SP001-06-DOSTAJ000,4,1,2015-07-23T17:20:00,2015-08-29T11:30:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-DOSTAJ-00351,40.13408,-70.77013,,101,134,Profiling body detached and on surface; recovered by SOOP
+SI20150722,,MGL1512,,CP01CNSP-SP001-07-SPKIRJ000,4,1,2015-07-23T17:20:00,2015-08-29T11:30:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-SPKIRJ-00265,40.13408,-70.77013,,101,134,Profiling body detached and on surface; recovered by SOOP
+SI20150722,,MGL1512,,CP01CNSP-SP001-08-CTDPFJ000,4,1,2015-07-23T17:20:00,2015-08-29T11:30:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-CTDPFJ-00347,40.13408,-70.77013,,101,134,Profiling body detached and on surface; recovered by SOOP
+SI20150722,,MGL1512,,CP01CNSP-SP001-09-FLORTJ000,4,1,2015-07-23T17:20:00,2015-08-29T11:30:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-FLORTJ-01205,40.13408,-70.77013,,101,134,Profiling body detached and on surface; recovered by SOOP
+SI20150722,,MGL1512,,CP01CNSP-SP001-10-PARADJ000,4,1,2015-07-23T17:20:00,2015-08-29T11:30:00,CGMCP-01CNSP-00004,CGVEH-CSPPCP-00005,CGINS-PARADJ-00437,40.13408,-70.77013,,101,134,Profiling body detached and on surface; recovered by SOOP
```

Comment:
Looks good - matches updates noted on metadata deploycsv_review google sheet.  Note - there are a couple updates needed for sensor cals - will capture those in slack channel.

Reply:
Looks good.